### ISSUE TITLE
showPrompt fails 

### ISSUE THREAD

**Issue by serafeimgr** [steelbrain/package-deps#103]
Hello,
showPrompt flag fails on the newest Atom 1.17.0-beta5
Rolled back into stable 1.16 and it works as expected.

**Comment by steelbrain** [comment]
Any code or steps to reproduce this would be nice and help us debug/fix this faster

**Comment by serafeimgr** [comment]
Let me rephrase because the title is wrong.
showPrompt work as expected.
The loading bar does not have the same behavior as in 1.16

**Comment by Arcanemagus** [comment]
Are you still seeing this @serafeimgr? If so, please provide some GIFs showing what you mean as this package seems to work just fine for everyone else.

**Comment by serafeimgr** [comment]
Yes, the visualization (animation) is not the same anymore and does not depict the actual installation progress.

Sorry if i mislead you in the first place. 
For example.
1.19 beta
![image](https://user-images.githubusercontent.com/17761449/28800323-e13ca1b4-7654-11e7-9f78-89199fb0d7ca.png)

Behavior before 1.17
https://cloud.githubusercontent.com/assets/4278113/22874527/59b37c22-f1e8-11e6-968e-dfa857db7664.png

**Comment by steelbrain** [comment]
That would be the behavior of this line is returning null which it shouldn't be https://github.com/steelbrain/package-deps/blob/26c062bae2859fec4d127c5928d05b5296d5f776/src/view.js#L29

Could it be that your notifications package is taking too long to activate? (Invoke `Timecop: View` to find out)

**Comment by serafeimgr** [comment]
It seems that this is the root cause, even though sometimes the packages installation fails with no apparent reason.
Anyway, you should close this issue, i will try to find a workaround :)

Thanks,
Serafeim

**Comment by steelbrain** [comment]
Good luck!
